manager/manager/types.go

```diff
@@ -84,7 +84,7 @@ func NewDeployment(name string, id int) *Deployment {
 // DeploymentStatus is an enumeration type for the status of a deployment.
 type DeploymentStatus string
 
-// These constants implement the deploymentStatus enumeration type.
+// These constants implement the DeploymentStatus enumeration type.
 const (
 	CreatedStatus  DeploymentStatus = "Created"
 	DeletedStatus  DeploymentStatus = "Deleted"
```

Comment:
Never mind. I see that these are deployment status, while the others are resource status. Makes sense.